test/integration/data-types.test.js

```diff
@@ -56,14 +56,18 @@ describe(Support.getTestDialectTeaser('DataTypes'), () => {
     });
   });
 
-  const testSuccess = function(Type, value) {
+  const testSuccess = function(Type, value, useBindParam) {
```

Comment:
Third argument should be options with arbitrary configuration possibilities ``{ useBindParam }``